web/src/lib/roadmap/buildUserRoadmap.ts

```diff
@@ -162,6 +162,30 @@ const getIndustryBasedAddOns = (profileData: ProfileData, industryId: string | u
     }
   }
 
+  if (profileData.industryId === "employment-agency") {
+    if (profileData.employmentPersonnelServiceType === "JOB SEEKERS") {
+      addOns.push("employment-agency-job-seekers");
+    }
+
+    if (profileData.employmentPersonnelServiceType === "EMPLOYERS") {
+      switch (profileData.employmentPlacementType) {
+        case "TEMPORARY": {
+          addOns.push("employment-agency-employers-temporary");
+          break;
```

Comment:
dust - are the break statement necessary?

Reply:
yes its is recommended